ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -273,6 +274,10 @@ private[state] object Conversions {
           tx,
         ),
     )
+  }
+
+  def parseTransaction(rawTx: Raw.Transaction): TransactionOuterClass.Transaction =
```

Comment:
```suggestion
  private[this] def parseTransaction(rawTx: Raw.Transaction): TransactionOuterClass.Transaction =
```

Reply:
Can we just inline it?

Reply:
Cannot change it to `private[this]` as this was being used outside. I thought it would indicate places that ideally we'd like to move the `kv-transaction-support` library. I don't think it actually helps, so probably the best idea is to inline it, which I did.